server/src/main/java/org/opensearch/search/fetch/subphase/highlight/HighlightPhase.java

```diff
@@ -37,6 +37,7 @@
 import org.opensearch.common.regex.Regex;
 import org.opensearch.index.mapper.KeywordFieldMapper;
 import org.opensearch.index.mapper.MappedFieldType;
+import org.opensearch.index.mapper.MatchOnlyTextFieldMapper
```

Comment:
Missing semicolon. (Showing up in the build output at https://build.ci.opensearch.org/job/gradle-check/52567/console.)

If you run `./gradlew precommit`, it should catch that and any code formatting issues.

Thanks!

Reply:
Nevermind -- looks like you already caught it 😁

Reply:
what a shame... really sorry about that

Reply:
Too much Python recently has dulled my Java 🙃